Dangerfile

```diff
@@ -36,8 +36,8 @@
 @S_LICENSE_FILES ||= ["LICENSE"]
 
 # set the patterns to watch and warn about if they need security review
-@S_SECURITY_FILE_PATTERN ||= /Dangerfile|auth|login|permission|email|account|admin|twofactor|sudo/
-@S_SECURITY_CONTENT_PATTERN ||= /auth|login|password|permission|token|secret|security|scope|key|sudo/
+@S_SECURITY_FILE_PATTERN ||= /Dangerfile|(auth|login|permission|email|account|admin|twofactor|sudo).*\.py/
```

Comment:
I'm not sure what you're going for, but this will catch and like, `auth*.py`, etc. Does this pattern get applied to directories, or literally the filenames? Like, woudl this cover say, `auth/foo.py` or only `auth.py` ? Hopefully the full path.